### PR TITLE
attempt to make hover for garden

### PR DESCRIPTION
**Pull request by kokopuffz**
not sure how to get plant info on hover

### CODE REVIEW

**Comment by uzorjchibuzor** [comment]
> not sure how to get plant info on hover

You mean nothing is showing up when you hover over the target <div> ?

**Comment by kokopuffz** [comment]
i just want to access the plant info o.o

**Comment by uzorjchibuzor** [comment]
Okay you mean the plant info to be shown on the hovercard itself? Let me pull the branch and look closely. Gimme some minutes.

**Comment by uzorjchibuzor** [comment]
The card flipping is a very nice aesthetic concept, while it take a bit of a time to load the content of the hover card (we can work on this later).

File: app/assets/stylesheets/application.tailwind.css [hunk 1]
```diff
@@ -48,4 +48,42 @@
...
+    background-color: transparent;
+  }
+
+  .flip-content {
+    position: relative;
+    width: 100%;
+    height: 100%;
+    text-align: center;
+    transition: all 1s;
+    transform-style: preserve-3d;
```
Comment (uzorjchibuzor) [inline app/assets/stylesheets/application.tailwind.css:63]: On the next issue, can you try `overflow-y: scroll` if it would work?

**Review by uzorjchibuzor — APPROVED** [review]
This is a very nice job. look into the comment I made if it is possible but this LGTM.

File: app/assets/stylesheets/application.tailwind.css (hunk 1 above)
Comment (kokopuffz) [reply app/assets/stylesheets/application.tailwind.css:63]: will do!

**Pull request merged**